src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/instance_blackrock_depths.cpp

```diff
@@ -573,11 +583,17 @@ class instance_blackrock_depths : public InstanceMapScript
                         boss->SetLootRecipient(nullptr);
                     }
                     boss->setFaction(FACTION_FRIEND);
+                    if (i ==  6) // doomrel needs explicit reset
```

Comment:
double space

Reply:
pls remove it

Reply:
should be ok in new commit, waiting for CI build.